integration_tests/projects/008_pure_python_models/scripts/before.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+model_name = context.current_model.name
+
+output = model_name
+
+path = os.path.join(os.environ["temp_dir"], model_name + ".before.txt")
+with open(path, "w") as file:
+    file.write(output)
```

Comment:
Let's start using the `_fal_testing.create_*` stuff (with the `prompt="before"`)
https://github.com/fal-ai/fal/blob/4373278c33c8b34214a434d127cb82d2de1b3a96/tests/_fal_testing/utils.py#L22-L23